docs/virtual_machines/istio_service_mesh.md

```diff
@@ -0,0 +1,144 @@
+# Istio service mesh
+
+Service mesh allows to monitor, visualize and control traffic between microservices.
```

Comment:
s/microservices/pods/ ? 

My point is that a VM runs in a pod - and is not a microservice.

Reply:
That's a good point